map/traffic_manager.cpp

```diff
@@ -344,8 +353,10 @@ void TrafficManager::CheckCacheSize()
       auto const it = m_mwmCache.find(mwmId);
       if (it->second.m_isLoaded)
       {
-        m_currentCacheSizeBytes -= it->second.m_dataSize;
+        // Note. It's necessary to multiply by two because routing and rendering use separate caches.
+        m_currentCacheSizeBytes -= 2 * it->second.m_dataSize;
```

Comment:
|m_currentCacheSizeBytes| should be increased when the cache is grown up and decreased when the cache is clear decreased. So |m_currentCacheSizeBytes|  is increased in OnTrafficDataResponse() and decreased here. If I increase something multiplied by 2 I have to decrease it multiplied by 2.